### DISCUSSION TITLE
Virtuelles RaspberryMatic und CCU1

### DISCUSSION THREAD

**Discussion by matzeeg3** [OpenCCU/OpenCCU#1335] [🇩🇪 Fragen&Antworten] [answered]
Moin in die Runde, ich plane nun Step by Step die Umstellung von meiner CCU1 auf eine Virtuelle RaspberryMatic. Nun habe ich folgende frage: Kann ich mein e "alte CCU1" also LAN Gateway für Funk und Wired Komponenten nutzen?
Das wäre ja 1a :)
Weiß jemand was.
Vielen dank.
Gruß
Matze

**Comment by jens-maus** [comment, marked as answer]
Nein, die CCU1 ist zu alt und nur noch was für die Tonne! Die kannst du weder als LAN-RF Gateway noch als LAN-Wired Gateway an RaspberryMatic anbinden.

**Reply by matzeeg3** [reply]
ok danke für die Antwort, wie komme ich den vom "Lan-RF" Gateway ans Wired System?

**Reply by matzeeg3** [reply]
Wenn ich es bislang richtig verstanden habe, brauche ich nun 2 komponenten 1x das LAN-RS Gateway und einmal das LAN-Wirless Gateway
